src/components/Dissolution/AssociationDetails.vue

```diff
@@ -194,12 +194,12 @@ export default class AssociationDetails extends Mixins(CommonMixin, DateMixin) {
 
   /** The entity name. */
   get entityName (): string {
-    return this.getBusinessLegalName || GetCorpNumberedDescription(this.getEntityType)
+    return this.getBusinessLegalName || GetCorpNumberedDescription(this.getEntityType as any)
```

Comment:
There's a mismatch between CorpTypeCd in GetCorpNumberedDescription() and CorpTypeCd from getEntityType that I couldn't figure out quickly so I just typed it as any.